.travis.yml

```diff
@@ -12,15 +16,26 @@ env:
 notifications:
   email: false
 
-install:
+before_cache:
+  - rm -rf $HOME/miniconda/locks $HOME/miniconda/pkgs $HOME/miniconda/var
+  - rm -rf $HOME/miniconda/conda-bld $HOME/miniconda/conda-meda/history
+
+before_install:
   # We use the older version of conda because there is a bug in version 4.3.27.
-  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda2-4.3.21-Linux-x86_64.sh -O miniconda.sh;
-    else
-      wget https://repo.continuum.io/miniconda/Miniconda3-4.3.21-Linux-x86_64.sh -O miniconda.sh;
+  - if [[ ! -d "$HOME/miniconda/bin" ]]; then
+      rm -rf $HOME/miniconda;
+      mkdir -p $HOME/download;
+      if [[ -d "$HOME/download/miniconda.sh" ]]; then rm -rf $HOME/download/miniconda.sh; fi;
```

Comment:
do we need this line?

Reply:
I test to remove this line

Reply:
Travis passed. I remove this line in current PR